### PR TITLE
Add scripts and workflows for CI/CD. Chocolatey deploy support added.

### PR DESCRIPTION
**Pull request by mtirionMSFT**
* Added GitVersion setup for versioning based on the repo.
* Added PowerShell based build and pack scripts that can be run locally as well.
* Added nuspec definition and scripts for Chocolatey package creation.
* Added Build & Test workflow being triggered on push to any branch
* Added manual triggered Release & Publish workflow. Will created a release in the repo and publish to Chocolatey
* Modified the README to indicate how to install using Chocolatey

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,17 @@ This repository contains a series of tools, templates, tips and tricks to make y
...
 * [DocLanguageTranslator](./src/DocLanguageTranslator): allows to generate and translate automatically missing files or identify missing files in multi language pattern directories.
 
+## Install
+
+The tools can be installed by dowloading the zip-file of a [release](https://github.com/Ellerbach/docfx-companion-tools/releases) or use [Chocolatey](https://chocolatey.org/install) like this:
+
+```shell
+choco install docfx-companion-tools
+```
+
```
Comment (Ellerbach) [inline README.md:18]: Good, maybe how to run them here as well

File: changelog-config.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "categories": [
+    {
+      "title": "## 🚀 Features",
+      "labels": ["feature", "enhancement"]
```
Comment (Ellerbach) [inline changelog-config.json:5]: Good idea! Please make sure the labels are documented somewhere!

File: build.ps1 [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Include
```
Comment (Ellerbach) [inline build.ps1:1]: Maybe for all scripts add a small header

**Review by Ellerbach — APPROVED** [review]
Nice work!

File: README.md (hunk 1 above)
Comment (mtirionMSFT) [reply README.md:55]: Added more details how to run after install in README.

File: build.ps1 (hunk 3 above)
Comment (mtirionMSFT) [reply build.ps1]: All scripts have a header now and explain what they do.

**Pull request merged**